std/process.d

```diff
@@ -3155,11 +3163,17 @@ unittest
     assertThrown(environment["std_process"]);
 
     // get() without default value
-    assert (environment.get("std_process") == null);
+    assert (environment.get("std_process") is null);
 
     // get() with default value
     assert (environment.get("std_process", "baz") == "baz");
 
+    version(Posix) {
```

Comment:
There is code in the windows implementation to specifically return null in the case the environment variable isn't found.

Windows command prompt is not the only user of environment variables.